sdk/storage/Azure.Storage.Queues/src/QueueClient.cs

```diff
@@ -1374,6 +1441,80 @@ private void SetNameFieldsIfNull()
         #endregion DeleteMessage
 
         #region UpdateMessage
+        /// <summary>
+        /// Changes a message's visibility timeout and contents. The message content must be a UTF-8 encoded string that is up to 64KB in size.
+        /// For more information, see <see href="https://docs.microsoft.com/en-us/rest/api/storageservices/update-message"/>.
+        /// </summary>
+        /// <param name="messageId">ID of the message to update.</param>
+        /// <param name="popReceipt">
+        /// Required. Specifies the valid pop receipt value returned from an earlier call to the Get Messages or Update Message operation.
+        /// </param>
+        /// <param name="messageBytes">
+        /// The message bytes.
+        /// </param>
+        /// <param name="visibilityTimeout">
+        /// Required. Specifies the new visibility timeout value, in seconds, relative to server time. The new value must be larger than
+        /// or equal to 0, and cannot be larger than 7 days. The visibility timeout of a message cannot be set to a value later than the
+        /// expiry time. A message can be updated until it has been deleted or has expired.
+        /// </param>
+        /// <param name="cancellationToken">
+        /// Optional <see cref="CancellationToken"/>.
+        /// </param>
+        /// <returns>
+        /// <see cref="Response{UpdatedMessage}"/>.
+        /// </returns>
+        public virtual Response<UpdatedMessage> UpdateMessage(
```

Comment:
Similar general feedback here, we aren't passing strings.

Reply:
Fixed